spyder_terminal/widgets/terminalgui.py

```diff
@@ -108,6 +122,15 @@ def setup_term(self):
             dict_options[option] = self.get_conf(option)
         self.apply_settings(dict_options)
         self.apply_zoom()
+        shell_name = self.get_conf('shell')
+        if os.name != 'nt':
```

Comment:
What happens for Windows? Can envs be activated without problems there?

Reply:
Yes, they work as it is